### PR TITLE
Add Credential Management and Payment Request to PWA checklist

### PR DESCRIPTION
**Pull request by owencm**
Adding these features to the PWA checklist as requirements to hit the 'showcase' bar, if using them is appropriate for the use case, in accordance to internal discussion with sabineb and agektmr.

Note these are not required for the baseline bar.

I've not squashed these as I hear GitHub has a new squashing capability when accepting PRs, but let me know if you'd like them squashed on my end.

### CODE REVIEW

File: src/content/en/progressive-web-apps/checklist.md [hunk 1]
```diff
@@ -851,3 +851,71 @@ Note: This check list only applies if notifications are implemented
...
+             Create an account for a service and ensure you see the save 
+             password/account dialog show up. Click on "Save".
+          </li>
+          <li>Clear cookies for the site (via clicking on the padlock or Chrome
+              settings) and refresh the site. Ensure that you either see an 
+              account picker (e.g. if there are multiple accounts saved) or 
+              are automatically signed back in.
+          </li>
+          <li>
+            Sign out and refresh the site. Ensure you see the account picker.
```
Comment (jpmedley) [inline src/content/en/progressive-web-apps/checklist.md]: 'Ensure that you see the account picker.'

File: src/content/en/progressive-web-apps/checklist.md [hunk 2]
```diff
@@ -851,3 +851,71 @@ Note: This check list only applies if notifications are implemented
...
+  </tbody>
+</table>
+
+Note: This check only applies if your site accepts payments
+
+<table class="responsive">
+  <tbody>
+    <tr>
+      <th colspan=2><h3>
+        User can pay easily via native UI from Payment Request API
```
Comment (jpmedley) [inline src/content/en/progressive-web-apps/checklist.md]: Missing period.

File: src/content/en/progressive-web-apps/checklist.md [hunk 3]
```diff
@@ -851,3 +851,71 @@ Note: This check list only applies if notifications are implemented
...
+      </h3></th>
+    </tr>
+    <tr>
+      <td><b>To Test</b></td>
+      <td>
+        Enter the payment flow. Instead of filling out a conventional form, 
+        verify the user is able to pay easily via the native UI provided by the
```
Comment (jpmedley) [inline src/content/en/progressive-web-apps/checklist.md]: 'triggered' might be a better verb than 'provided'. The later implies a level of control the Payment Request API doesn't have.

**Comment by jpmedley** [comment]
@owencm The link should actually be /web/progressive-web-apps/checklist

File: src/content/en/progressive-web-apps/checklist.md [hunk 4]
```diff
@@ -851,3 +851,71 @@ Note: This check list only applies if notifications are implemented
...
+          <li>
+            Sign out and refresh the site. Ensure you see the account picker.
+          </li>
+          </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><b>To Fix</b></td>
+      <td>
+        Follow our [Credential Management API Integration Guide](https://developers.google.com/web/fundamentals/security/credential-management/)
```
Comment (petele) [inline src/content/en/progressive-web-apps/checklist.md]: For internal links, use only the absolute path, not the fully qualified URL. This prevents links from breaking on the China site (developers.google.cn) and other domains we're hosted on.

**Comment by owencm** [comment]
Fixed above feedback. Let me know if there's anything else!

**Comment by owencm** [comment]
Just reviewed in production, it seems the links I introduced somehow didn't get parsed correctly as links, anyone have an idea about what went wrong?

**Comment by petele** [comment]
Yep, found it after I merged. DevSite doesn't permit markdown embedded in HTML :(

I submitted #4022 which fixes that, [cleaned up a bit of the language, fixed a few other links](https://github.com/google/WebFundamentals/pull/4022/files) and [promotes the checklist on the landing page](https://web-central.appspot.com/web/progressive-web-apps/). 

I'll likely do another push later today or first thing (ET) tomorrow.

**Comment by petele** [comment]
Live.

**Comment by owencm** [comment]
My hero 😍

**Pull request merged**